src/main/java/org/sagebionetworks/template/etl/EtlBuilderImpl.java

```diff
@@ -22,14 +23,16 @@
 import static org.sagebionetworks.template.Constants.EXCEPTION_THROWER;
 import static org.sagebionetworks.template.Constants.GLUE_DATABASE_NAME;
 import static org.sagebionetworks.template.Constants.JSON_INDENT;
-import static org.sagebionetworks.template.Constants.PROPERTY_KEY_INSTANCE;
+import static org.sagebionetworks.template.Constants.PROPERTY_KEY_GLUE_DATA_BASE_NAME;
 import static org.sagebionetworks.template.Constants.PROPERTY_KEY_STACK;
+import static org.sagebionetworks.template.Constants.S3_KEY;
 import static org.sagebionetworks.template.Constants.STACK;
 import static org.sagebionetworks.template.Constants.TEMPLATE_ETL_GLUE_JOB_RESOURCES;
 
 public class EtlBuilderImpl implements EtlBuilder {
 
-    public static final String GLUE_DB_NAME = "synapsewarehouse";
+    public static final String PROCESSED_ACCESS_RECORD_FOLDER_NAME = "processedAccessRecord";
```

Comment:
There shoulnd't be any specific job property in this file, this needs to come from a configuration